javascript/javascript1/week2/eventapplication.js

```diff
@@ -0,0 +1,17 @@
+function getEventeekday(inDays) {
+  const weekDays = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  // getting todays day from system
+  const todays = new Date().getDay();
+  const idx = (todays + inDays) % weekDays.length;
+  return weekDays[idx];
```

Comment:
everything is fine here. Very nice and clean. You could return a message with your function , but not really necessary.